pkg/apiserver/endpoints/handlers/get.go

```diff
@@ -120,7 +119,7 @@ func ListResourceWithOptions(r cm_rest.ListerWithOptions, scope handlers.Request
 		trace.Step("Listing from storage done")
 
 		responsewriters.WriteObjectNegotiated(scope.Serializer, negotiation.DefaultEndpointRestrictions, scope.Kind.GroupVersion(), w, req, http.StatusOK, result)
-		trace.Step(fmt.Sprintf("Writing http response done"))
```

Comment:
:sweat_smile: it is always fun to see a decent linter going through a code after a while